bee-block/src/output/mod.rs

```diff
@@ -218,11 +218,11 @@ impl Output {
     }
 
     /// Verifies if a valid storage deposit was made. Each [`Output`] has to have an amount that covers its associated
-    /// byte cost, given by [`ByteCostConfig`].
+    /// byte cost, given by [`RentStructure`].
```

Comment:
Should this also be rent cost?
```suggestion
    /// rent cost, given by [`RentStructure`].
```